modules/gis/src/lib/binary-to-geojson.js

```diff
@@ -94,7 +97,7 @@ function parseFeature(data, startIndex, endIndex) {
 
 /** Parse input binary data and return an object of properties */
 function parseProperties(data, startIndex, endIndex) {
-  const properties = Object.assign(data.properties[data.featureIds.value[startIndex]]);
+  const properties = Object.assign({}, data.properties[data.featureIds.value[startIndex]]);
```

Comment:
You should be able to remove the `Object.assign` altogether? If `data.properties` is an array of objects.

Reply:
Yes I think so, I've added this before fixing the properties from `geojsonToBinary`. Thanks!